GroupIS1024/Migrations/20221113202147_model-changes.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Any idea what this is?

Reply:
Professor, Migration should be applied after data model changes.

https://learn.microsoft.com/en-us/ef/core/managing-schemas/migrations/?tabs=dotnet-core-cli